Formula/eiffelstudio.rb

```diff
@@ -17,20 +17,32 @@ class Eiffelstudio < Formula
   depends_on "pkg-config" => :build
   depends_on "gtk+"
 
+  uses_from_macos "pax" => :build
+
   def install
-    system "./compile_exes", "macosx-x86-64"
-    system "./make_images", "macosx-x86-64"
+    # Fix flat namespace usage in C shared library.
+    if OS.mac?
+      system "tar", "xf", "c.tar.bz2"
+      inreplace "C/CONFIGS/macosx-x86-64",
+        "-flat_namespace -undefined suppress",
+        "-undefined dynamic_lookup"
```

Comment:
```suggestion
      inreplace "C/CONFIGS/macosx-x86-64",
                "-flat_namespace -undefined suppress",
                "-undefined dynamic_lookup"
```
Though I'd put this in one line if it meets the line limit.